src/components/BookCard.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] [OPTIONAL] Kindly avoid disabling Eslint because it is supposed to help/guide the standards of our Javascript code.  I am leaving this as an optional recommendation because you only disabled it once.